src/levanter/models/llama.py

```diff
@@ -305,17 +305,20 @@ def update_state_dict(self, state_dict: StateDict, prefix: Optional[str] = None)
         return state_dict
 
 
-class LlamaRMSNorm(hnn.LayerNorm):
-    """It is a modified version of LayerNorm.
-    The main changes are:
-    1. The variance is defined as the average of square, versus the original
-    definition as the average of the squared deviations from the mean.
-    2. The output is defined as x * inv, without minusing the mean.
-    3. The default value of eps is set to 1e-6 and use_bias to False.
+class LlamaRMSNorm(eqx.Module):
     """
+    Similar to LayerNorm, but uses the RMS of the input along the specified axis (or axes) instead of variance.
+    """
+
+    axis: AxisSpec = eqx.static_field()
+    weight: Optional[NamedArray]
+    bias: Optional[NamedArray]
+
+    eps: float = eqx.static_field(default=1e-5)
+    dtype: Optional[jnp.dtype] = eqx.static_field(default=jnp.float32)
 
     @staticmethod
-    def init(axis: AxisSpec, eps: float = 1e-6, use_weight: bool = True, use_bias: bool = False):
+    def init(axis: AxisSpec, eps: float = 1e-5, use_weight: bool = True, use_bias: bool = True, dtype=jnp.float32):
```

Comment:
What's your rationale behind these changes to the default value?

Llama's implementation doesn't have bias, right?
https://github.com/huggingface/transformers/blob/main/src/transformers/models/llama/modeling_llama.py#L88

The default rms eps is 1e-6: 
https://huggingface.co/docs/transformers/en/model_doc/llama2#transformers.LlamaConfig

Reply:
copypasta